main/libadwaita/template.py

```diff
@@ -38,3 +38,10 @@
 @subpackage("libadwaita-devel")
 def _devel(self):
     return self.default_devel()
+
+
+@subpackage("libadwaita-demo")
+def _demo(self):
+    self.pkgdesc = f"{pkgdesc} (demo applications)"
+    return ["usr/bin/adwaita-1-demo"]
```

Comment:
```suggestion
    return [
        "usr/bin/adwaita-1-demo",
        "usr/share",
    ]
```

it has other files too